recipes-extended/pktgen-dpdk/pktgen-dpdk_19.12.0.bb

```diff
@@ -12,9 +12,9 @@ S = "${WORKDIR}/git"
 
 DPAA_VER ?= "dpaa"
 export RTE_TARGET = "arm64-${DPAA_VER}-linuxapp-gcc"
-export RTE_SDK = "${RECIPE_SYSROOT}/usr/share/"
+export RTE_SDK = "${RECIPE_SYSROOT}${datadir}/dpdk"
```

Comment:
seems to interfere with another commit. In such a case, better re-order or squash together.